bootstrap/_rpm_common.sh

```diff
@@ -20,11 +20,15 @@ $tool install -y \
   git-core \
   python \
   python-devel \
-  python-virtualenv \
-  python-devel \
+  python-setuptools \
   gcc \
   dialog \
   augeas-libs \
   openssl-devel \
   libffi-devel \
   ca-certificates \
+
+if ! $tool install -y python-virtualenv ; then
+    easy_install pip
+    pip install virtualenv
```

Comment:
@centminmod I'm not certain if we should drop Python 2.6 supports, because there's probably many old LTS distributions do not have Python 2.7 in their official repos.